decode/evm_rpc.go

```diff
@@ -89,6 +89,20 @@ func MethodHasBlockHashParam(method string) bool {
 	return includesBlockHashParam
 }
 
+var StaticMethods = []string{
```

Comment:
doc comment on what makes a method static

Reply:
also need to update / expand the caching dev docs with details on the caching strategies and different categories of methods with a list of what strategies are used for which sets of methods

Reply:
done